core/ui.libs/dbcp.json

```diff
@@ -39,6 +40,7 @@
     "versions": {
       "1.0.0": "QmcMekTfYhhWLvnHo3YwSkw5fWo77FNHLrqgLfmcBYCerN",
       "1.1.0": "QmWGG5kwUJfgoHRRbJYNNAnE32mWLvRYH3Fc288v2CJaNb",
+      "1.10.0": "QmcK9yFh5YnciVXFavsFRVTWaYTHN4fd6rxGDtztfmG7hA",
```

Comment:
I think alphabetic sorting doesn't apply here, was correct before.

Reply:
anyways shall we use a new major version here since we have breaking API changes?

Reply:
Jeha completly correct. Don't know, why i updated this version. Must be a mistake.